src/main.rs

```diff
@@ -55,7 +64,10 @@ fn symbolic_revert(sub_matches: &ArgMatches) -> Result<(), String> {
 
     let query = yools::encoder::encode_revert_unreachable::<EVMInstructions>(&ast);
 
-    match solver::query_smt(&query) {
+    match solver::query_smt_with_solver(
+        &query,
+        solver::SolverConfig::new(sub_matches.value_of("solver").unwrap_or("cvc4")),
```

Comment:
Shouldn't we rather use `default_value` when setting up the options?

Reply:
Whats that?

Reply:
ah you mean directly in the cli options

Reply:
pushed it